packages/component-meta/scripts/generateComponentMeta.ts

```diff
@@ -6,16 +6,21 @@ import { TAG_NAMES, INTERNAL_TAG_NAMES } from '@porsche-design-system/shared';
 import type { TagName } from '@porsche-design-system/shared';
 
 const glue = '\n\n';
+
+global.ROLLUP_REPLACE_IS_STAGING = 'staging';
+
 // can't resolve @porsche-design-system/components without building it first, therefore we use relative path
 const sourceDirectory = path.resolve('../components/src/components');
 const componentFiles = globby.sync(`${sourceDirectory}/**/*.tsx`);
 
-const getExportFilePath = (source: string, constName: string, tagName: TagName): string => {
+const getComponentFilePath = (tagName: TagName): string => {
+  return componentFiles.find((file) => file.match(new RegExp(`${tagName.replace(/^p-/, '/')}\\.tsx$`)));
+};
```

Comment:
can be shortened:

```suggestion
const getComponentFilePath = (tagName: TagName): string =>
  componentFiles.find((file) => file.match(new RegExp(`${tagName.replace(/^p-/, '/')}\\.tsx$`)));
```

Reply:
Yea it can but that's not improving a consistent code style.